src/module/canvas/token/aura/renderer.ts

```diff
@@ -97,29 +106,40 @@ class AuraRenderer extends PIXI.Graphics implements TokenAuraData {
     /** Draw the aura's texture, resizing the image/video over the area (applying adjustments to that if provided) */
     async #drawTexture(): Promise<void> {
         const data = this.appearance.texture;
-        if (!data || this.token.isPreview || this.texture) {
+        if (!data || this.token.isPreview || this.textureContainer) {
             return;
         }
 
-        const maybeTexture = await loadTexture(data.src, { fallback: "icons/svg/hazard.svg" });
-        this.texture = maybeTexture instanceof PIXI.Texture ? maybeTexture : null;
-        if (!this.texture) return;
-
-        const globalVideo = isVideoFilePath(data.src) ? game.video.getVideoSource(this.texture) : null;
-        if (globalVideo) {
-            this.texture.destroy();
-            this.texture = await game.video.cloneTexture(globalVideo);
-            const video = game.video.getVideoSource(this.texture) ?? globalVideo;
-            video.playbackRate = data.playbackRate;
-            const offset = data.loop ? Math.random() * video.duration : 0;
-            game.video.play(video, { volume: 0, offset, loop: data.loop });
-        }
+        const texture = await (async (): Promise<PIXI.Texture | null> => {
+            const maybeTexture = await loadTexture(data.src, { fallback: "icons/svg/hazard.svg" });
+            if (!(maybeTexture instanceof PIXI.Texture)) return null;
+
+            const globalVideo = isVideoFilePath(data.src) ? game.video.getVideoSource(maybeTexture) : null;
```

Comment:
given maybeTexture is getting destroyed anyways, would it not be better to make the entire image texture load part of what happens when there is no video texture?

Reply:
This didn't change in the PR and was pulled from core. For some reason it's necessary?

Reply:
I misread the code, getVideoSource is pulled from maybeTexture already.